src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -1087,6 +1088,10 @@ namespace GraphQL.Builders
     {
         protected FieldBuilder(GraphQL.Types.FieldType fieldType) { }
         public GraphQL.Types.FieldType FieldType { get; }
+        public virtual GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> ApplyMiddleware(GraphQL.Instrumentation.IFieldMiddleware middleware) { }
+        public virtual GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> ApplyMiddleware(System.Func<GraphQL.IResolveFieldContext, GraphQL.Instrumentation.FieldMiddlewareDelegate, System.Threading.Tasks.ValueTask<object?>> middleware) { }
```

Comment:
This overload differs in its signature from `schema.FieldMiddleware.Use(Func<FieldMiddlewareDelegate, FieldMiddlewareDelegate> transform)`